README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# To Do list: list structure
```

Comment:
- [ ] Kindy remove the awesome book description on line 14 also the live demo link on line 98 since they are not part of the project.
- [ ] [optional] Kindy add how to run your project locally in the getting started section, and remove headers that don't have content eg live demo deployment.